### ISSUE TITLE
Remove belittling words from all docs in the Backdrop codebase

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#3351]
## Describe your issue or idea

In an attempt to make the code more welcoming to newcomers and coding beginners, we should remove phrases like "just", "of course", "simply", "easily", "obviously" from our in-code documentation. 

- [x] of course (only instances are in LICENSE.txt)
- [x] obviously: https://github.com/backdrop/backdrop/pull/2346
- [x] simply: https://github.com/backdrop/backdrop/pull/2345
- [x] easily: https://github.com/backdrop/backdrop/pull/2362
- [ ] just

Based on the fantastic idea from Symfony:
https://twitter.com/symfonydocs/status/1056912807034265600

**Comment by herbdool** [comment]
There are a few other small changes in this PR not mentioned, such as removing space before the top docblock. But it all looks fine.

**Comment by jenlampton** [comment]
> ...such as removing space before the top docblock

Yeah, I try to make that change along with all the others I'm doing. It's not the kind of improvement that's worth an issue of it's own, but if I'm in a file anyway I try to take care of it :) You'll see that a lot from me!

**Comment by quicksketch** [comment]
Thanks, I merged all the changes in the available PRs. As this is already 3 major instances, I'm going to close this issue. Please open a new issue for any further replacements.

**Comment by quicksketch** [comment]
These were merged both into 1.x and 1.11.x btw, so this will be in 1.11.3.

**Comment by jenlampton** [comment]
There's a few more words we should remove from the codebase (just, and some others) so I'd like to leave this open for a bit.

**Comment by jenlampton** [comment]
no final PR here, removing milestone.

**Comment by BWPanda** [comment]
Stil no PR, removing milestone candidate.
